EC-84: Restructure eZ Commerce doc, part 2 (#2)
ezsystems/commerce-documentation: docs/guide/bestseller/bestseller_cookbook.md

# Bestseller cookbook

## Add a bestseller slider to the Landing Page

Add bestseller block from the Elements overlay.
    
Drag the Bestseller block to the desired place and drop it.
    
![](../img/bestseller_1.jpg)
    

> Edit the added block by clicking on the gear and select either a category of eZ products or add the Category code of an eContent category:

Reviewer: ```suggestion
Edit the added block: click the gear, then either select a category of eZ products or add the Category code of an eContent category:
```
